src/nixfeature.cc

```diff
@@ -30,8 +30,9 @@ namespace nixfeature {
         nix::Feature currFeat = input.entity<nix::Feature>(1);
         //TODO properly implement link type
         struct_builder sb({ 1 }, { "linkType" });
-        sb.set("linkType");
+	std::string s = "linkType";
+        sb.set(s);
```

Comment:
Good point: My build is indeed 4 days old.
I will try

Am 02/21/2015 um 04:07 PM schrieb Christian Kellner:
>> @@ -30,8 +30,9 @@ namespace nixfeature {
>>          nix::Feature currFeat = input.entity<nix::Feature>(1);
>>          //TODO properly implement link type
>>          struct_builder sb({ 1 }, { "linkType" });
>> -        sb.set("linkType");
>> +	std::string s = "linkType";
>> +        sb.set(s);
> 
> Do you have a (very) recent NIX installed on the system? Because this will if `nix::Value`'s char constructors are not marked explicit.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/G-Node/nix-mx/pull/34/files#r25122681
>